### PR TITLE
DBS Mitschriften: Entity Relationship Model

### PR DESCRIPTION
**Pull request by Steve2955**
Mitschriften zur zweiten Vorlesung

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- DBS-DORENDORF/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: DBS-DORENDORF/README.md [hunk 1]
```diff
@@ -81,3 +85,48 @@ Ableiten einer Informationsstruktur $\rightarrow$ Abbildung von Objekten und Bez
...
+
+![eindeutige Beziehung](assets/erd-relationship-1-1.png)<!--width=400px-->
+
+![einseitig eindeutige Beziehung](assets/erd-relationship-1-n.png)<!--width=400px-->
+
+![komplexe Beziehung](assets/erd-relationship-m-n.png)<!--width=400px-->
+
+**Attribute (Eigenschaten, Merkmale)**
+
+- beschrieben Eigenschaften der einzelner Objekte oder Beziehungen
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:118]:
```suggestion
- beschreiben Eigenschaften der einzelner Objekte oder Beziehungen
```

**Review by RvNovae — APPROVED** [review]
Kleiner typo ansonsten supi.

Danke!

**Review by ZeroPointMax — APPROVED** [review]
mh, irgendwie habe ich ein Deja vu...

Danke fürs Mitschreiben 👍

**Pull request merged**